uk.co.compucorp.civicrm.hrcontactactionsmenu/templates/CRM/ContactActionsMenu/Page/Inline/ActionsPart2.tpl

```diff
@@ -0,0 +1,122 @@
+<div id="crm-contact-actions-list">
+  <div id="bootstrap-theme">
+    <div class="row">
+      <div class="col-md-4 contact_action_menu__user_info">
+        <div class="row">
+          <h3>User Informations:</h3>
+          <p><strong>User:</strong> <a href="#" class="text-primary"> 100 Lauren Ipsum </a></p>
+          <p><strong>Roles:</strong> <span class="grey-text"> civihr_staff, civihr_manager </span></p>
+          <p>
+            <button type="button" class="btn btn-primary-outline">
+              <span class="btn-icon">
+                <i class="fa fa-envelope-o"></i>
+              </span> 
+              Send Welcome Email
+            </button>
+          </p>
+          <p>
+            <button type="button" class="btn btn-primary-outline">
+              <span class="btn-icon">
+                <i class="fa fa-envelope-o"></i>
+              </span> 
+              Send Password Reset Email
+            </button>
+          </p>
+          <h3>User Has Access To:</h3>
+          <p>
+            <span class="grey-text">No staff members</span> 
+            <span class="fa fa-question-circle help-icon"></span>
```

Comment:
`help-icon` is a custom class. Should be named as `crm_contact_action_menu__help_icon`

Reply:
Please check all other custom classes.